lib/Exposed.ml

```diff
@@ -59,7 +59,9 @@ module Right = struct
     | {pcd_args= args; _} -> constructor_arguments args
 
   let type_declaration = function
-    | {ptype_attributes= _ :: _; _} -> false
+    | { ptype_attributes= {attrs_before= _ :: _; _} | {attrs_after= _ :: _; _}
```

Comment:
This is written several time and is not clear. I think we need a `Ast.Ext_attrs.is_empty`.